server/service/endpoint_setup.go

```
@@ -48,13 +49,19 @@ func makeSetupEndpoint(svc kolide.Service) endpoint.Endpoint {
 		if err != nil {
 			return setupResponse{Err: err}, nil
 		}
+		// If everything works to this point, log the user in and return token
+		_, token, err = svc.Login(ctx, *req.Admin.Username, *req.Admin.Password)
+		if err != nil {
+			return setupResponse{Err: err}, nil
```

returning an error here could be problematic because it would break the form and the setup endpoint would just get shut off anyway since the user is already created. 

I would ignore the error here(it will be logged by the service), and just omit the token and force them to log in if we somehow fail to create a session.

---

@groob nice catch